zerver/lib/export.py

```diff
@@ -1336,6 +1337,7 @@ def import_uploads_local(import_dir: Path, processing_avatars: bool=False) -> No
     with open(records_filename) as records_file:
         records = ujson.loads(records_file.read())
 
+    re_map_foreign_keys(records, 'realm_id', related_table="realm", id_field=True)
```

Comment:
`re_map_foreign_keys` will fix the outdated realm_id in `records.json` with the new allocated one.